### ISSUE TITLE
Windows compilation: Document how to workaround Antimalware Service slowdown

### ISSUE THREAD

**Issue by akien-mga** [godotengine/godot-docs#2973]
I've been using MinGW on Windows 10 a bit the past few days, and every time I ran a build I noticed that Windows 10's Antimalware Service Executable can eat between 40 to 80% of my CPU cycles (!) on an i7 8705G (building with `-j7`).

This is easy to workaround by adding a Folder exception to one's dev/build folder, so it might be worth adding a note about it in our docs, as I think this can cause huge slowdowns for people compiling Godot: https://support.microsoft.com/en-us/help/4028485/windows-10-add-an-exclusion-to-windows-security

I haven't checked if MSVC builds have the same issue - it might be that Antimalware Service Executable already has an exception rule for `cl.exe` (MSVC) but not for `cc1plus.exe` (GCC). (CC @Calinou)

We should phrase the tip in a way that doesn't give users the impression that we want to expose their system to malwares though :)

**Comment by RaTcHeT302** [comment]
no, windows defender hammers the compiler, i added expections for it and IT STILL FREAKING SCANS EVERYTHING, it's driving me insane

**Comment by RaTcHeT302** [comment]
i turned defender off, and is it targetting this file?
https://i.imgur.com/s4N3ylG.png

ntoskrnl.exe
